agent/pkg/models/models.go

```diff
@@ -16,6 +16,14 @@ func GetEntry(r *tapApi.MizuEntry, v tapApi.DataUnmarshaler) error {
 	return v.UnmarshalData(r)
 }
 
+type EntriesRequest struct {
+	LeftOff   int    `form:"leftOff" validate:"required,min=0,max=2147483647"`
```

Comment:
why the lefoOff is required?

Reply:
`leftOff` is the index the querying and fetching will start. That's why it's required. I forgot that it can take a negative value as well so I set min to `-1` with https://github.com/up9inc/mizu/commit/3d88ae84f41a1c8060bac0395f39de6f9c6f7760